### PR TITLE
Set id via payload, to allow 1:1 ids from id.wmo

### LINKED ISSUE #47: Change user id field to bigint
It's bigserial right now, but to keep a 1:1 mapping with id.wmo user ids, we should have it be non auto-incrementing.

### PR DESCRIPTION
**Pull request by cadecairos**
Fixes #47

### CODE REVIEW

File: services/api/routes/public.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ var routes = [
     config: {
       validate: {
         payload: {
+          id: numericSchema.required(),
```
Comment (jbuck) [inline services/api/routes/public.js]: Joi converts strings to numbers by default, so I think this can just become `Joi.number().integer().postive()`

**Comment by jbuck** [comment]
@cadecairos r+, but this needs a followup bug where it pulls data from the validated token, not from the user controlled payload

File: services/api/routes/public.js (hunk 1 above)
Comment (cadecairos) [inline services/api/routes/public.js]: postgreSQL `bigints` can get to be larger than the JS number type supports, so the numericSchema covers our butts should we ever encounter that problem by using:

```
Joi.alternatives().try(
  Joi.number().integer(),
  Joi.string().regex(/^\d+$/)
);
```

I can add in a positive rule as well.

Comment (jbuck) [inline services/api/routes/public.js]: Ah, clever. Nevermind then!

**Pull request merged**